sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/CreateTableExec.scala

```diff
@@ -42,7 +44,12 @@ case class CreateTableExec(
       tableSpec.properties, tableSpec.options, tableSpec.serde,
       tableSpec.location, tableSpec.comment, tableSpec.provider,
       tableSpec.external)
-    CatalogV2Util.withDefaultOwnership(props)
+    val newProps = props.get(TableCatalog.PROP_LOCATION).map { loc =>
+      props + (TableCatalog.PROP_LOCATION ->
```

Comment:
Can we be consistent and always qualify the path in `DataSourceV2Strategy`?

Reply:
en. I do it.